subcommands/status

```diff
@@ -1,12 +1,34 @@
 #!/usr/bin/env bash
 
-if [ -e $(soji note-path log) ]; then
-  task=`cat $(soji note-path log) | egrep "^\*\* (start|break|lunch|meditation|meeting)" | tail -1 | sed 's/[^-]*-- //' | sed 's/\(.*\) --.*/\1/' |  awk -F '' '{if (NF > 20) {print substr($0, 0, 21)"..."} else print $0}'`
-  task=`echo $task | xargs`
-  start_time=`cat $(soji note-path log) | egrep "^\*\* (start|break|lunch|meditation|meeting)" | tail -1 | sed 's/.*-- //'`
-  if [ "$start_time" != "" ]; then
-    seconds=`expr $(date +'%s') - $(date +'%s' --date $start_time)`
-    minutes=`expr $seconds / 60`
-    echo "$task -- $minutes min"
-  fi
+if [ "$os_x" != "" ]; then
```

Comment:
Yes, I actually discovered that this morning. My ~/.config/soji/subcommands had an outdated build on my machine from when I was just focused on getting the OS/X issues worked out. That is corrected in the current build.